src/main/resources/db/migration/V1_78__move_appointments_to_new_action_plan_sessions.sql

```diff
@@ -0,0 +1,65 @@
+create table action_plan_session_appointment_pre_v1_78 as select * from action_plan_session_appointment;
+
+with action_plans_to_modify as (
+    with referrals_with_multiple_approved_aps as (
+        select
+            r.id referral_id
+        from
+            referral r
+        where
+            r.id in (
+                     '05b1d52a-ae5c-49a6-8b4d-080a5101e79f',
```

Comment:
You are right, however I see it as a safety check to prevent updating anything that we shouldn't. 

My understanding is that even if we make it repeatable for other referrals - it will only ever be run once because it's part of migration.

I originally had the first part as: 

```
        select
            ap.referral_id
        from
            referral r
                inner join action_plan ap on ap.referral_id = r.id
        where
            ap.approved_at is not null
          and (r.concluded_at is null and r.end_requested_at is null)
        group by
            ap.referral_id
        having
                count(ap.referral_id) = 2
```

would you prefer that?

Reply:
Yes, that's why I think this shouldn't be a versioned migration, but more like a job we can run on demand

Reply:
@rajivmahal1 What are you thoughts regarding hard-coded vs dynamic?

I guess it conflicts with your comment.